sql/bene_monthly_audit.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS public.beneficiary_monthly_audit;
+
+CREATE TABLE public.beneficiary_monthly_audit
```

Comment:
yes and now....open to suggestions.
YES, it would be useful in subsequent audit activities (i.e., join vs. beneMonthly when looking at new vs chg'd). Also it would prevent duplicate rcds of same bene_id + yearMonth.
NO, while it is highly unlikely, if we did hit a case where we hit a unique constraint violation, then the whole job craps out and a rollback occurs...
But I think I agree that a primary key would be beneficial here from the get-go (was thinking of indexing the table post-run, but might as well do it early).

Reply:
I actually like things crapping out. Wouldn't that mean there was something funky going on with the data or the code?